buildSrc/src/main/kotlin/Constants.kt

```diff
@@ -37,4 +37,4 @@ object Constants {
         const val minSdkVersion = 19
```

Comment:
We should consider going to higher than 19, that's ridiculously old
https://apilevels.com/